### PR TITLE
Add isort to pre-commit

### PR DESCRIPTION
**Pull request by ki4070ma**
It's to keep imports clean.

### CODE REVIEW

File: ci.sh [hunk 1]
```diff
@@ -7,5 +7,12 @@ if [[ $result ]] ; then
   exit 1
 fi
 
+result=$(python -m isort -rc . | grep -v "Skipped")
+if [[ $result ]] ; then
```
Comment (mykola-mokhnach) [inline ci.sh]: what do we check here?

Comment (ki4070ma) [reply ci.sh]: Checking if unordered imports aren't remained

Comment (mykola-mokhnach) [reply ci.sh]: shouldn't we check the exit code instead (`$?`) ?

Comment (ki4070ma) [reply ci.sh]: I've checked its behavior at first, but unfortunately both cases(ordered, unordered) retrun `0`.

Comment (mykola-mokhnach) [reply ci.sh]: how about https://stackoverflow.com/questions/2480584/how-do-i-use-a-file-grep-comparison-inside-a-bash-if-else-statement ?

Comment (ki4070ma) [reply ci.sh]: Will look into again.
At least I found  below issue and codes.

https://github.com/timothycrosley/isort/issues/423
https://github.com/timothycrosley/isort/blob/d093f97c1d26854fbb082e43cfae8a062dacda56/isort/main.py#L382

Comment (ki4070ma) [reply ci.sh]: I could do as expected by using ` --check-only` opt for `isort` :)

**Pull request merged**